src/pcedit/pc.menus.mac.mm

```diff
@@ -65,7 +65,7 @@ void init_menubar() {
 	
 	MenuHandler* handler = [[[MenuHandler alloc] init] retain];
 	setMenuCallback([apple_menu itemWithTitle: @"About BoE Character Editor"], handler, @selector(menuChoice:), int(eMenu::ABOUT));
-	setMenuCallback([apple_menu itemWithTitle: @"Preferences…"], handler, @selector(menuChoice:), int(eMenu::PREFS));
+	setMenuCallback([apple_menu itemWithTitle: @"Settings…"], handler, @selector(menuChoice:), int(eMenu::PREFS));
```

Comment:
…what are you doing? The menuitem should be Preferences, not Settings.

Reply:
The actual menu item that's in there is "Settings..." on Mac for both cases for me. Which is very confusing because I see in the `xib` files it should be "Preferences..."

But at actual runtime, my scenario editor and character editor menu items say Settings, not Preferences, and don't work (are disabled) unless these strings match.

Reply:
That's _really_ weird. That's not even the case for me – when I run the scenario editor, I see Preferences, not Settings.

Reply:
Could this be a scons vs xcode problem? Is the xcode build doing something with the xib file that scons isn't?

Reply:
I see in the scons build output:

> ibtool --compile build/obj/menus/scenedit.nib rsrc/menus/scenedit.xib
Install file: "build/obj/menus/scenedit.nib" as "build/Blades of Exile/BoE Scenario Editor.app/Contents/Resources/scenedit.nib"

So I should look at that nib file.

Reply:
It's not human-readable and Xcode isn't showing me the contents either.

Reply:
I'll revert the change and open an issue for it.